simple-counter/src/lib.rs

```diff
@@ -1 +1,157 @@
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::serde::{Deserialize, Serialize};
+use near_sdk::{near_bindgen, AccountId};
 
+#[near_bindgen]
+#[derive(BorshDeserialize, BorshSerialize)]
+pub struct State {
+    count: u64,
+    auth_ids: Vec<AccountId>,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct Transaction {
+    from: AccountId,
+    value: u64,
+}
+
+#[near_bindgen]
+impl State {
+    #[init]
+    pub fn new(val: u64, auth: Vec<AccountId>) -> Self {
+        State {
+            count: val,
+            auth_ids: auth,
+        }
+    }
```

Comment:
```suggestion
    pub fn new(val: u64, auth: Vec<AccountId>) -> Self {
        let mut this = Self{
            count: val,
            auth_ids: UnorderedSet::new(StorageKey::AuthAccount),
        };

        for auth_id in auth {
            this.auth_ids.insert(&auth_id);
        }

        this
    }
```
이 처럼 구현하면, `UnorderedSet` 의 요구사항을 만족하면서 구현 할 수 있을 것 같습니다. `StorageKey::AuthAccount` 는 위쪽에 따로 정의해주면 될 것 같습니다. 

```rust
#[derive(BorshStorageKey, BorshSerialize)]
pub enum StorageKey {
    AuthAccount,
}
```

[Reference](https://github.com/near-examples/nft-series/blob/e674b75391580b6e5707da59ab0116d3a81c4077/market/src/lib.rs)

Reply:
자주 사용하게 될 패턴으로 보이네요. 좋은 refer 공유 감사합니다 반영했습니다.